examples/models/h2o_example/H2OModel.py

```diff
@@ -1,3 +1,4 @@
+from __future__ import print_function
```

Comment:
We are happy removing Python 2.x support completely, so there is no need for the `__future__` imports.

Reply:
Imports from __future__ should be removed from everywhere, not added.